app/mailers/subscription_mailer.rb

```diff
@@ -54,10 +54,14 @@ def notify_tag_added(node, tag, tagging_user)
     )
    end
 
-  def send_digest(user_id, top_picks)
-    subject = "Your weekly digest"
+  def send_digest(user_id, top_picks, k)
```

Comment:
Can we give this parameter a more informative name, so people understand when reading the code? Thanks!

Reply:
Also, maybe top picks could just become "nodes", you know?

Reply:
Yes absolutely, and I am gonna rename top_picks parameter to nodes then?